cloud2butt.py

```diff
@@ -19,13 +19,12 @@ def getResponseHeadersAndBody(self, content):
   def processHttpMessage(self, tool, is_request, content):
     if is_request:
       return
-    headers, body = self.getResponseHeadersAndBody(content)
```

Comment:
@minanagehsalalma I think this line is still needed? You could then do `if body is not None` below and parse the results.

Reply:
Hmm the error is in that line
You can't get headers abd body if the request is empty(none)

Where would it get the needed data from?

It has to check the availability of the data first before trying to fetch to not cause this error

So... M

Reply:
Sorry i forgot to the line even that it's in the version of the extension i am using 

I added it now , Hopefully it's now fixed